src/test/regress/sql/subselect.sql

```diff
@@ -1069,3 +1069,21 @@ drop table tl1;
 drop table tl2;
 drop table tl3;
 drop table tl4;
+
+--Test case for subquery, which returns more than one rows
+-- start_ignore
+drop table if exists table1, table2;
+-- end_ignore
+
+create table table1 as
+    select * from (values (1, 0), (1, 0)) v(a, b) distributed by (a);
+create table table2 as
+    select * from (values (0, 10), (0, 10)) v(a, b) distributed by (a);
+
+explain (costs off)
+select * from table1 where 10 in
+    (select b from table2 where table2.a = 0 or table1.b = table2.b);
+select * from table1 where 10 in
+    (select b from table2 where table2.a = 0 or table1.b = table2.b);
```

Comment:
What about adding empty lines between multiline queries as it was done in the existing code? It will be easier to understand where query ends and the next one begins.